source/mainnet/net/resources/glossary.rst

```diff
@@ -585,3 +606,10 @@ Winning probability
 The winning probability is the probability that a baker wins in a given slot.
 The probability is :math:`1-(1-f)^α`, where :math:`f` is the difficulty parameter and :math:`α` is
 the :ref:`lottery power<glossary-lottery-power>`.
+
+.. _glossary-zero-knowledge-proof:
+
+Zero-knowledge proof
+====================
+
+A method by which a user (the prover) can prove to another party (the verifier) that the user meets a requirement without revealing anything beyond the truth of the proven statement.
```

Comment:
This mixes two ways of stating things. "requirement" and "proven statement".

Perhaps this would be better.
```
A method by which a user (the prover) can prove to another party (the verifier) that the user meets a requirement without revealing anything beyond that.
```